torch/_subclasses/fake_tensor.py

```diff
@@ -442,18 +461,25 @@ def from_tensor(self, tensor):
         with no_dispatch():
             return self.fake_tensor_converter(self, tensor)
 
-
 def run_cpu_fallback(func, args, kwargs, orig_not_implemented_exception):
     with no_dispatch():
         def to_cpu(e):
             if isinstance(e, FakeTensor):
-                return torch.zeros_like(e, device="cpu")
+                return torch.ones_like(e, device="cpu")
             return e
 
         try:
             args = tree_map(to_cpu, args)
             kwargs = tree_map(to_cpu, kwargs)
 
+            # TODO: full decomposition/fix
```

Comment:
Add note that this is a hack due to no meta coverage. But isn't there already a meta-function for `aten._embedding_bag`?

Reply:
There's not, there is `_embedding_bag_forward_only`